src/rust-2018/documentation/new-editions-of-the-book.md

```diff
@@ -6,6 +6,8 @@
 
 ![Minimum Rust version: 1.28](https://img.shields.io/badge/Minimum%20Rust%20Version-1.28-red.svg) for drafts of the 2018 edition
```

Comment:
Why are the rust versions noted for **draft** versions of the books? Can they not just be removed?

Reply:
Sounds reasonable to me.

Reply:
Thanks @ehuss!